src/content/tutorials/aws-lambda.mdx

```diff
@@ -7,66 +7,34 @@ id: aws-lambda
 
 import Image from '@theme/Image';
 
-This tutorial will show you how to deploy a development environment in Okteto that contains an AWS Lambda function with Okteto.
+This tutorial will show you how to deploy a development environment in Okteto that contains an AWS Lambda function with Okteto. We use [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam-overview.html) to create the Lambda function in this tutorial. You can find the code for the sample app we work with during this tutorial [here](https://github.com/okteto/aws-lambda).
+
+If you'd like to see an alternate approach using [Terraform](https://www.terraform.io/) instead of AWS SAM CLI, checkout this [sample app](https://github.com/okteto-community/aws-lambda-with-terraform).
 
 ## Prerequisites
 
 - Access to an Okteto instance
 - Access to an AWS account with the permissions to create IAMs
 
-## Step 1: Create an IAM user
+## Step 1: Configure Cloud Credentials
 
-The IAM user that you use with AWS SAM must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+Okteto needs sufficient permissions to make necessary AWS service calls and manage AWS resources to create the Lambda Function in your AWS account. This can be done  by [configuring Cloud Credentials](admin/cloud-credentials/aws.mdx).
 
-The following permissions will enable you to complete this tutorial:
+Make sure to give the following permissions to the IAM role you create when configuring Cloud Credentials:
 
 - AWSCloudFormationFullAccess
 - IAMFullAccess
 - AWSLambda_FullAccess
 - AmazonAPIGatewayAdministrator
 - AmazonS3FullAccess
 
-Once the user has been created, [generate a set of access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html), and save them file locally.
-
-In order for the installer to be able to use this keys, we need to add them as [Okteto Variables](core/okteto-variables.mdx). Use `AWS_ACCESS_KEY_ID` as the name for the variable that contains the access ID, and `AWS_SECRET_ACCESS_KEY` as the name for the variable that contains the access key.
```

Comment:
Should we leave this section as a backup in case someone doesn't want to use Cloud Credentials? I can't see why they wouldn't but is it worth showing another option?

Reply:
In general, I find it confusing when docs give me multiple options. That being said, using credentials it's a lot easier than roles tbh, so I can see how some people will prefer that. Maybe we can add it as an info block @codyjlandstrom ?

Reply:
Maybe lets remove it for now and create this as a community post? That may be the best spot for this information, then we don't clog up the onboarding guides